service/policy/unsafe/unsafe.proto

```diff
@@ -186,6 +186,19 @@ message UnsafeDeleteAttributeValueResponse {
   policy.Value value = 1;
 }
 
+// WARNING!!
+message UnsafeDeletePublicKeyRequest {
```

Comment:
If delete is unsafe, is update also unsafe?

Reply:
Update only allows modification if `is_active`. Not sure if thats an unsafe action though.

Reply:
I removed the modification of is_active. I am thinking keys should be immutable.